Makefile.toml

```diff
@@ -102,8 +102,27 @@ cargo d
 [tasks.deadlink]
 script = ['''
 #!/usr/bin/env bash -eux
-cargo deadlinks --check-http --ignore-fragments
+mkdir -p target
+set +e
+# create deadlinks log file
+cargo deadlinks --check-http --ignore-fragments | tee target/cargo-deadlinks.out
+set -e
+# sort uniq for `cargo-deadlinks` output
+cat target/cargo-deadlinks.out | sort | uniq > target/cargo-deadlinks.sort
+# check diff for `deadlinks.expected`
+cat deadlinks.expected | sort | uniq > target/cargo-deadlinks.expected.sort
+diff -u target/cargo-deadlinks.expected.sort target/cargo-deadlinks.sort
+
 mlc --ignore-path target
+
+''']
+
+[tasks.deadlink-expect]
```

Comment:
want to depend on this task from `deadlink` task?

Reply:
No, this task seeds output expectation to `deadlink.expected`

Reply:
Ah, sorry. Got it.